ocf-spec/0.0.1/schema/interface.json

```diff
@@ -68,10 +68,25 @@
                 "type": "object",
                 "description": "The input parameters for a given Action.",
                 "properties": {
+                  "typeRef": {
+                    "$ref": "https://capact.io/schemas/common/type-ref.json"
+                  },
                   "jsonSchema": {
```

Comment:
Why you haven't removed it?

Reply:
This is for input parameters. In some cases we don't need to create dedicated type. Sometimes is better to inline jsonschema as we do not want to create TI, and we also don't reuse it anywhere, so to reduce the boilerplate you can just put directly in Interface manifest.